classes/studnt.rb

```diff
@@ -6,6 +6,11 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.student.push(self) unless classroom.student.includes?(self)
+  end
+
   def play_hooky
     '¯(ツ)/¯'
   end
```

Comment:
- Hi, Can you fix a typo mistake here?  change the file name from `studnt.rb` to `student.rb`